lib/rubycritic/cli/options.rb

```diff
@@ -19,6 +19,16 @@ def parse
             @root = path
           end
 
+          opts.on('-b', '--branch BASE_BRANCH,FEATURE_BRANCH', 'Set branches to compare') do |branches|
+            self.base_branch, self.feature_branch = branches.split(',').map { |branch| String(branch) }
```

Comment:
what do you think of raising an error if the two branch names are the same?
Since it will obviously not produce any useful result, and will run the analysers twice